src/site/content/en/learn/css/box-model/index.md

```diff
@@ -0,0 +1,196 @@
+---
+layout: course
+title: Box Model
+subhead: This post highlights a few powerful lines of CSS that do some serious heavy lifting and help you build robust modern layouts.
+authors:
+  - andybell
+tags:
+  - css
+---
+
+## Learn CSS - Box Model
+
+Say you have this bit of HTML:
+
+```html
+<p>I am a paragraph of text that has a few words in it.</p>
+```
+
+Then say you wrote this CSS for it:
+
+```css
+p {
+  width: 100px;
+  height: 50px;
+  padding: 20px;
+  border: 1px solid;
+}
+```
+
+The content would break out of your element and it would be 140px wide, rather than 100px. Why is that? The box model is a core foundation of CSS and understanding how it works, how it is affected by other aspects of CSS and importantly, how you can control it will help you to write more predictable CSS.
+
+<iframe height="500" style="width: 100%;" scrolling="no" title="Learn CSS - Box model intro" src="https://codepen.io/piccalilli/embed/0c03f5dc8a83ccaa2f48fcc115e1ec54?height=500&theme-id=38982&default-tab=result," frameborder="no" loading="lazy" allowtransparency="true" allowfullscreen="true">
```

Comment:
Hey @robdodson I've just dumped the embed iframes in for now but a shortcode that'll allow us to determine height and tabs would be excellent.

Reply:
Tracking this here: https://github.com/GoogleChrome/web.dev/issues/4801

If you have existing code that you're OK sharing then we could use that, otherwise if you know the interface for how you'd like it to work then I could build something off of that.

Reply:
PR is here, https://github.com/GoogleChrome/web.dev/pull/4842